<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Print_SSCC</class>
 <widget class="QWidget" name="Print_SSCC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Печать SSCC для агрегации</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Количество SSCC кодов:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spnNumber">
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblWork">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="prgBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnPrint">
        <property name="text">
         <string>Печать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetup">
        <property name="text">
         <string>Настройки принтера</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
